### PR TITLE
Allow each instance of enrichment to fetch data from a different SQS

### PR DESCRIPTION
**Pull request by commuterjoy**
- [x] Set environment variables before merging
- [x] Default non-production environments to using the debug queue
- [x] Make the health end points aware of both regions (fastly & cloud watch)

/cc @akilananb @tom-parker 

This was the simplest thing I could come up with. It was too complex to have a single process fetching data from multiple queues.

So, in effect, I'm just parameterizing the configuration,

```
node --harmony-destructuring server/enrichment.js --region us
```

Final push ...

## Weds

- [x] Shut down deployments on spoor-enrichment
- [x] Grant permissions to AWS accounts to CloudWatch & get metrics working
- [ ] Akilan to do health checks for Spoor-API
- [x] Investigate instability in pipeline
- [x] Fix the SNS unpacking issue

## Thurs

- [ ] Stop (or scale down) enrichment on the old stack
- [ ] Scale up new stack to 100%
- [ ] Test spoor-consumer works with new stack
- [ ] Tell everyone to move over in the afternoon (1pm !!!)

### CODE REVIEW

File: server/enrichment.js [hunk 1]
```diff
@@ -8,30 +8,26 @@ const Pipeline= require('./pipelines');
 const metrics = require('next-metrics');
 const sinks = require('./sinks');
 const reportError = require('./utils/report-error');
+const argv = require('minimist')(process.argv.slice(2));
 
 metrics.init({ app: 'spoor-enrichment', flushEvery: 30000 });
```
Comment (tom-parker) [inline server/enrichment.js]: Is it worth adding the region in here too so that we can break down all the metrics for both?

**Comment by tom-parker** [comment]
:+1:  looks good

File: server/enrichment.js (hunk 1 above)
Comment (commuterjoy) [inline server/enrichment.js]: yup, https://github.com/Financial-Times/next-metrics/blob/master/lib/metrics.js#L60 does this automatically

Comment (tom-parker) [inline server/enrichment.js]: Won't that always be EU for this though, independent of which process it is? Or am I missing something

Comment (commuterjoy) [inline server/enrichment.js]: oh, right - got it - yes, good point.

Comment (commuterjoy) [inline server/enrichment.js]: hah, actually, no, https://github.com/Financial-Times/next-metrics/blob/master/lib/metrics.js#L56

takes the dyno name, which will be `sqseu` or `sqsus`

Comment (tom-parker) [inline server/enrichment.js]: Ok sold!

File: server/utils/report-error.js [hunk 2]
```diff
@@ -14,6 +14,6 @@ client.capture = function (name, err, level) {
 }
 
 // If any global occurance
-client.patchGlobal();
+// client.patchGlobal();
```
Comment (commuterjoy) [inline server/utils/report-error.js]: FIXME - only apply this != production

**Pull request merged**